crmprtd/bc_hydro/download.py

```diff
@@ -0,0 +1,116 @@
+''' Downloads data from BC hyrdo
+
+BC Hydro posts a rolling window (3 months) observing hourly data
+once a week. Data is in txt files.
+
+It is recommended to run this script once every 3 months to collect
+all available data, supplemented with aggressive monitoring and
+alerting for errors. If the script is run less than 3 months you
+will miss data.
+'''
+
+import pysftp
+from tempfile import TemporaryDirectory
+import logging
+import os
+import sys
+import re
+from zipfile import ZipFile
+from argparse import ArgumentParser
+from datetime import date, timedelta
+
+from crmprtd import logging_args, setup_logging, common_auth_arguments
```

Comment:
Do we actually *use* `common_auth_arguments` for anything anymore? I think that this can be pulled.

Reply:
I think we need it for username